alerts/README.md

```diff
@@ -2,15 +2,58 @@
 title: Alerts
 ---
 
-This plugin extension contains [Prometheus Alertmanager](https://github.com/prometheus/alertmanager) via [prometheus-operator](https://github.com/prometheus-operator/prometheus-operator) and [Supernova](https://github.com/sapcc/supernova), the holistic alert management UI.
+Learn more about the **alerts** plugin. Use it to activate Prometheus alert management for your Greenhouse organisation. 
 
-# Owner
+The main terminologies used in this document can be found in [core-concepts](https://cloudoperators.github.io/greenhouse/docs/getting-started/core-concepts).
 
-1. Richard Tief (@richardtief)
-2. Tommy Sauer (@viennaa)
-3. Martin Vossen (@artherd42)
 
-### alerts alertmanager parameters
+## Overview
+
+This Plugin includes a preconfigured _Prometheus Alertmanager_, which is deployed and managed via the _Prometheus_ Operator, and _Supernova_, an advanced user interface for Prometheus Alertmanager. In addition, certificates are automatically generated to ensure _Prometheus_ authentication sending alerts to the Alertmanager as well as Slack notification templates are provided.
+
+Components included in this Plugin:
+
+- [Prometheus Alertmanager](https://prometheus.io/docs/alerting/alertmanager)
+- [Supernova](https://github.com/cloudoperators/juno/tree/main/apps/supernova)
+
+Optionally, [Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator) can be used to manage _Prometheus_ and _Alertmanager_ instances. Usually, **kube-monitoring** Plugin installs _Prometheus Operator_.
+
+## Note
+
+It is not meant to be a comprehensive package that covers all scenarios. If you are an expert, feel free to configure the plugin according to your needs.
+
+The Plugin is a depth configured [kube-prometheus-stack](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/README.md) Helm chart which helps to keep track of versions and community updates.
```

Comment:
```suggestion
The Plugin is a deeply configured [kube-prometheus-stack](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/README.md) Helm chart which helps to keep track of versions and community updates.
```

What would you like to transport here? Not sure what `depth configured` means here.

Reply:
The `kube-prometheus stack` is a generic Helm chart that needs to be tailored to our needs. That's what I was trying to convey.